cmd/bbolt/command_root.go

```diff
@@ -18,7 +18,7 @@ func NewRootCommand() *cobra.Command {
 
 	rootCmd.AddCommand(
 		newVersionCobraCommand(),
-		newSurgeryCobraCommand(),
+		newSurgeryCommand(),
```

Comment:
This makes sense. Should we rename the rest of the Cobra commands as a follow-up? I can work on it.

Reply:
> Should we rename the rest of the Cobra commands as a follow-up? I can work on it.

Yes, please

Reply:
Opened issue #731 / PR #732

Reply:
Thanks. Merged